xonsh/completers/path.py

```diff
@@ -124,10 +124,16 @@ def _env(prefix):
 
 
 def _dots(prefix):
+    complete_dots = builtins.__xonsh__.env.get("COMPLETE_DOTS", "matching").lower()
+    if complete_dots == 'never':
```

Comment:
```suggestion
    if complete_dots == "never":
```

We use `black` to autoformat all python files and it prefers double-quotes to single (unless they're nested)